keyboards/ergodox_ez/keymaps/lukaus/config.h

```diff
@@ -0,0 +1,20 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+#include "../../config.h"
```

Comment:
Could you remove this include?  It's not actually needed, because of how the makefile system handles the config files.